base_pom.xml

```diff
@@ -0,0 +1,498 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can we name it `parent-pom`? It is more maven way.

Reply:
Renamed base_pom.xml to parent-pom.xml